browser-test/src/applicant_application_index.test.ts

```diff
@@ -13,7 +14,7 @@ describe('applicant program index page', () => {
   const primaryProgramName = 'Application index primary program'
   const otherProgramName = 'Application index other program'
 
-  beforeAll(async () => {
+  beforeEach(async () => {
```

Comment:
**suggestion:** You should be able to keep this as Before All if you make the `ctx` creation on L12 to 

` const ctx = createTestContext(/* clearDb= */ false)`

Reply:
Oh good call - updated!